app/helpers/helpers.rb

```diff
@@ -0,0 +1,11 @@
+class Helpers
+  
+
+    def self.current_user(session)
+      @user = User.find(session[:user_session_id])
+    end
+
+    def self.is_logged_in?(session)
```

Comment:
I believe your helper methods should have access to the `session` object without having to pass it in as a parameter.